src/cli.rs

```diff
@@ -55,6 +58,28 @@ pub struct DumpOpts {
     pub font: String,
 }
 
+#[derive(Debug, Options)]
+pub struct HasTableOpts {
+    #[options(help = "print help message")]
+    pub help: bool,
+
+    #[options(help = "table to check for", meta = "TABLE")]
```

Comment:
Is there a way of making the short and long commands explicit in gumdrop? I was struggling to find where `-t` came from. 🤔

Reply:
Tbh I find it kind of surprising that they eagerly give you those precious short flags by default! 😬

Reply:
There is, but by default it uses the first letter of the field. These options are all scoped by sub-command, which allows them to have overlapping short flags if needed.